
# Applied Statistics
## Grainne Boyle


In [1]:
# I import the following libraries for various functionalities in my project:  

import math # Provides extensive mathematical functions.  
import itertools # Provides tools for creating iterators such as combinations() and permutations(). Helps you loop and combine things efficiently.  
import random # Used for generating random numbers and random selections 
import numpy as np # package for numerical computations and arrays
import matplotlib.pyplot as plt # Used for creating visualizations and plots

 

# Add some notes and links on your research of different import libraries and how you plan to use them in your project:


## Problem 1: Extending the Lady Tasting Tea

[Lady tasting tea](https://en.wikipedia.org/wiki/Lady_tasting_tea)
This project begins with a look at the lady tasting tea, a randomised experiment devised by Ronald Fisher. In the experiment, the lady is given eight cups of tea, four prepared by adding milk first and four by adding tea first. The lady's task is to correctly identify the four cups prepared by one method or the other. The null hypothesis is that the subject has no real ability to distinguish between the preparation method of the teas. The test statistic is a count of the number of successful attempts to select the four cups prepared by a given method. The distribution of possible number of successes, assuming the null hypothesis is true, can be computed using the number of combinations.

I used chatgpt to help format and render mathematical formulas in LaTex so that they display clearly in the markdown cells in this notebook. ​[LaTeX Math in Jupyter Documentation](https://jupyterbook.org/en/stable/content/math.html)

In the formula for combinations, 

$$
\binom{n}{k} = \frac{n!}{k!(n-k)!},
$$

we choose $k$ objects from $n$, where $n$ is the total number of cups and $k$ is the number of cups with milk poured first.

In the original experiment, the number of possible ways to choose 4 cups out of 8 is:

$$
\binom{8}{4} = \frac{8!}{4!(8-4)!} = 70
$$

Since only one of these combinations is completely correct, the probability of identifying all 4 correctly by chance is:

$$
P = \frac{1}{\binom{8}{4}} = \frac{1}{70} \approx 0.0143
$$

This means that the lady has a 1.4% chance of guessing correctly.

## To do, add some notes to explain the calculation above and factorials and also to explain the null hypothesis from research.



In [2]:
# In the continuation of this experiment, suppose we have 12 cups of tea in total — 8 poured tea first and 4 poured milk first.

# Number of cups of tea in total
no_cups = 12

# Number of cups of tea with milk in first 
no_cups_milk_first = 4

# Number of cups of tea with tea in first 
no_cups_tea_first = 8

# Number of ways that we can select cups with milk in first from a total of 12 cups
ways = math.comb(no_cups, no_cups_milk_first)

# Show the result
ways

495

In the continuation of this experiment  — we see the results

where, $n = 12$ and $k = 4$.

The number of ways to choose 4 cups with the milk in first from 12 cups is calculated as:

$$
\binom{12}{4} = \frac{12!}{4!(12-4)!}
$$

Evaluating this gives:

$$
\binom{12}{4} = 495
$$

So, there are 495 different possible combinations of 4 cups with the milk in first among 12 cups in total. 

As the total number of cups grow, the number of possible combinations increases.

The probability of identifying all 4 cups with the milk in first cups is then:

$$
P = \frac{1}{\binom{12}{4}} = \frac{1}{495} \approx 0.0020
$$

This means that the lady has a 0.20% chance of guessing correctly.




## Problem 2: Normal Distribution

## Problem 3: t-Tests

##  Problem 4: Anova

![abacus](img/statistics.jpg)

-----------------------------------
# END

